docs/_documentations/remote-deploying-codewind.md

```diff
@@ -35,67 +35,106 @@ For more information, select from the following IDEs:
 
 2. **Have an active Kubernetes context and log in to the cluster.** Codewind can run in OpenShift 3.11, OpenShift 4.3, OpenShift in IBM Public Cloud, standalone Kubernetes, and Kubernetes in Docker.
 
-3. **For Linux desktop, ensure your workstation is set up to use a Keyring.** An example of a Keyring on Linux is Secret Service. 
+3. **For Linux desktop, ensure that your workstation is set up to use a Keyring.** An example of a Keyring on Linux is Secret Service. 
 
-4. **Have a Git command line client installed.** To download the latest Git command line client, see [https://git-scm.com/download](https://git-scm.com/download).
+4. **Optional: Have a Git command line client installed.** To download the latest Git command line client, see [https://git-scm.com/download](https://git-scm.com/download).
 
-## 1. Clone the Codewind operator repository
+## 1. Clone or download the Codewind operator repository
 
 The Codewind operator helps with the deployment of Codewind instances in an Openshift or Kubernetes cluster. Installing the Codewind operator is usually performed by your system administrator. 
 
-Clone the Codewind operator repository, for example: 
+Download the Codewind operator from the [codewind-operator repository](https://github.com/eclipse/codewind-operator) or if you have Git installed, clone the `codewind-operator` repository, for example: 
```

Comment:
Comma after "[codewind-operator repository](https://github.com/eclipse/codewind-operator)" because the comma separates two independent clauses.

Reply:
Comma added.